homeassistant/components/mqtt/cover.py

```diff
@@ -287,15 +274,61 @@ def tilt_message_received(msg):
                     self._config[CONF_TILT_MIN],
                     self._config[CONF_TILT_MAX],
                 )
+        else:
+            _LOGGER.warning("Payload '%s' is not numeric", payload)
+
+    def _position_message_received(self, payload):
```

Comment:
Since this used only by `position_message_received`, do we really need to separate this? it makes it harder to follow the flow.